test/test_user_dict.py

```diff
@@ -320,7 +320,7 @@ def test_import_invalid_word(self):
     def test_startup_processing(self):
```

Comment:
上で出ていた修正のsuggestionです
```suggestion
    def test_ update_dict(self):
```

Reply:
修正しました。